packages/shared/README.md

```diff
@@ -3,3 +3,5 @@
 This local package contains utilities shared by the other packages.
 
 It is not meant to be published to npm.
+
+This package need to be built and it's export correctly set up in order to be used by the other packages in browser tests.
```

Comment:
Hm, are you sure? Maybe adding `../../shared` to `tsconfig.json` inside hub/inference woudl work?

Reply:
It didn't really worked. Actually I'm not confident about releasing any of this because I built the package (`pnpm pack`) and try to use it with the Create React App and I have errors about having node related imports

```
Failed to compile.

Module build failed: UnhandledSchemeError: Reading from "node:fs/promises" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.
ERROR in node:fs/promises
Module build failed: UnhandledSchemeError: Reading from "node:fs/promises" is not handled by plugins (Unhandled scheme).
Webpack supports "data:" and "file:" URIs by default.
You may need an additional plugin to handle "node:" URIs.
    at /tmp/my-app/node_modules/webpack/lib/NormalModule.js:835:25
    at Hook.eval [as callAsync] (eval at create (/tmp/my-app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Object.processResource (/tmp/my-app/node_modules/webpack/lib/NormalModule.js:832:8)
    at processResource (/tmp/my-app/node_modules/loader-runner/lib/LoaderRunner.js:220:11)
    at iteratePitchingLoaders (/tmp/my-app/node_modules/loader-runner/lib/LoaderRunner.js:171:10)
    at runLoaders (/tmp/my-app/node_modules/loader-runner/lib/LoaderRunner.js:398:2)
    at NormalModule._doBuild (/tmp/my-app/node_modules/webpack/lib/NormalModule.js:822:3)
    at NormalModule.build (/tmp/my-app/node_modules/webpack/lib/NormalModule.js:966:15)
    at /tmp/my-app/node_modules/webpack/lib/Compilation.js:1371:12
    at NormalModule.needBuild (/tmp/my-app/node_modules/webpack/lib/NormalModule.js:1257:26)
  ```
  
  I really don't understand because then I manually deleted all references to any node import in `node_modules/@huggingface/inference` but the bug is still there :upside_down_face:

Reply:
do those problems happen with the current version? We have problems with some CDNs like skypack

Reply:
opened #137 

It only impacts `@huggingface/hub` for now, it should definitely be fixed before adding this to `@huggingface/inference` which is the more impactful lib